src/wizard/wizard.scss

```
@@ -27,68 +26,98 @@
     color: t(iui-color-background-positive);
   }
 
+  // Completed steps
+  .iui-wizard-circle {
+    align-items: center;
+    border-radius: 50%;
+    display: flex;
+    width: $iui-l;
+    height: $iui-l;
+    justify-content: center;
+    overflow-wrap: break-word;
+    padding: 0;
+    position: relative;
+    text-align: center;
+    user-select: none;
+    transition: background-color $iui-speed-fast ease-out, border-color $iui-speed-fast ease-out, color $iui-speed-fast ease-out;
+    @include themed {
+      border: 1px solid t(iui-color-foreground-positive);
+      background-color: t(iui-color-background-1);
+      color: t(iui-color-foreground-positive);
+    }
+  }
+
+  // Current step
   &.iui-current {
+    font-weight: $iui-font-weight-semibold;
+    
     .iui-wizard-step-name {
       @include themed {
         color: t(iui-color-foreground-positive);
       }
-      font-weight: $iui-font-weight-semibold;
     }
 
     .iui-wizard-circle {
-      font-weight: $iui-font-weight-semibold;
       @include themed {
         color: t(iui-color-foreground-accessory);
         border-color: t(iui-color-background-positive);
         background-color: t(iui-color-background-positive);
       }
     }
 
-    .iui-wizard-track-content::before {
+    .iui-wizard-track-content::after {
       @include themed {
-        background-color: t(iui-color-foreground-positive);
+        background-color: t(iui-color-background-5);
       }
     }
   }
 
-  &.iui-completed {
-    &.iui-clickable .iui-wizard-circle {
-      @include iui-focus(iui-focus-positive-box-shadow);
-
-      &:hover {
-        cursor: pointer;
-        @include themed {
-          background-color: t(iui-color-background-positive);
-          border-color: t(iui-color-background-positive);
-          color: t(iui-color-foreground-accessory);
-        }
+  // Future steps
+  &.iui-current ~ .iui-wizard-step {
```

Probably shouldn't use name of the class explicitly here.

Try this:
```suggestion
  &.iui-current ~ & {
```

or
```suggestion
  &.iui-current ~ #{&} {
```

---

Good suggestion, I will go with `&.iui-current ~ &`.